scripts/build_test.py

```diff
@@ -774,6 +778,19 @@ def _mock_safe_delete_file(unused_filepath):
         self.assertTrue(os.path.isfile(
             'core/tests/data/third_party/js/third_party.min.js.map'))
 
+        self.assertLess(
+            os.path.getsize(
+                'core/tests/data/third_party/css/third_party.min.css'),
+            os.path.getsize('core/tests/data/third_party/css/third_party.css'))
+        self.assertLess(
+            os.path.getsize(
+                'core/tests/data/third_party/js/third_party.min.js'),
+            os.path.getsize('core/tests/data/third_party/js/third_party.js'))
+        self.assertGreater(
```

Comment:
OK, you don't need to assert this one, there isn't really anything sensible to compare it to. I think you're comparing apples to oranges here.

Reply:
Done